aireview/justfile

```diff
@@ -15,4 +13,45 @@ run:
     poetry run python src/aireview/main.py
 
 tests:
-    poetry run pytest
\ No newline at end of file
+    poetry run pytest
+
+prefect-login:
+    poetry run python prefect/deploy/login.py
+
+# See the help message by: just manage-block --help
+manage-block +args:
+    poetry run python prefect/deploy/manage_block.py {{ args }}
+
+# See the help message by: just manage-flow --help
+manage-flow +args: prefect-login
+    poetry run python prefect/deploy/manage_flow.py {{ args }}
+
+manage-flow-local +args: prefect-login
+    poetry run python prefect/deploy/manage_flow.py {{ args }}
+
+[private]
```

Comment:
**style**: Docker build commands are duplicated between docker-build and docker-build-local with only minor differences. Consider using a shared recipe with conditional platform argument.